### DISCUSSION TITLE
[SUGGESTION] Improvements in the CRUD editor - Tab attribute

### DISCUSSION THREAD

**Discussion by luigipanniello** [TiagoSilvaPereira/vemto-issues#142] [Ideas & Suggestions]
it would be interesting as a built-in functionality in the editor when working with complex tables

![Cattura](https://user-images.githubusercontent.com/16260623/110673385-02cbf100-81d1-11eb-8d9c-4871a5dc7131.JPG)

**Comment by TiagoSilvaPereira** [comment]
Hi @luigipanniello thank you for your suggestion
